lldb/unittests/SymbolFile/DWARF/DWARFDIETest.cpp

```diff
@@ -104,3 +105,61 @@ TEST(DWARFDIETest, ChildIteration) {
   DWARFDIE no_children_die(unit, die_child0);
   EXPECT_TRUE(no_children_die.children().empty());
 }
+
+TEST(DWARFDIETest, PeekName) {
+  const char *yamldata = R"(
+--- !ELF
+FileHeader:
```

Comment:
This test could be improved to handle the `DW_AT_specification` and `DW_AT_abstract_origin` case I mentioned above.